Podcast/SubscriptionsViewController.swift

```diff
@@ -62,8 +63,8 @@ class SubscriptionsViewController: ViewController, UICollectionViewDelegate, UIC
     
     func setupCollectionViewFlowLayout() -> UICollectionViewFlowLayout {
         let layout = UICollectionViewFlowLayout()
-        let cellWidth: CGFloat = 160.5
-        let cellHeight: CGFloat = 210
+        let cellWidth: CGFloat = 160.5/375 * view.frame.width
```

Comment:
can we precalculate these percentages ?? fr cellHeight and cellWidth (aka 0.428 for the first)